### ISSUE TITLE
Allow students to download a copy of the .Rmd with answers filled in

### ISSUE THREAD

**Issue by coatless** [rstudio/tutor#19]
Perhaps I'm overthinking this, but would it be possible to have the students retain a copy of their work (outside of the saved session) via a .Rmd file that mimcs the form?

The closest I can think is to have them just print the form page. But, I would like them to be able to interact with it at a later time.

**Comment by jjallaire** [comment]
One problem with this is that exercises can have hidden setup code associated with them. If we include the setup code this might be confusing to students, if we don't include the setup code then the exported Rmd won't run.

The event recorder is another place where student work can be recorded for later review.

**Comment by garrettgman** [comment]
+1

I'm not sure what the best way forward is, but should we start using tutorials in in person workshops (I would like to), it would be useful for students to retain some record of their work, even if it is just an HTML version of the final output.

This would be worth doing well, because it can lay the groundwork for using tutorials as assignments, where students create something they then submit.

**Comment by apreshill** [comment]
+1 this was my students' biggest complaint when using learnr in the classroom. It feels like your work is lost and many took to copying/pasting code chunks into a local Rmd file.

**Comment by jjallaire** [comment]
I agree, it seems like this is a very high priority item based on feedback I've heard from many sources. @schloerke or @trestletech is this something we could take a look at doing soon?
